assets/.babelrc

```diff
@@ -1,5 +1,4 @@
 {
-    "presets": [
-        "@babel/preset-env"
-    ]
+  "presets": ["@babel/preset-env"],
+  "plugins": ["@babel/transform-runtime"]
```

Comment:
happen to know if/by how much this increases the `js` build output?

Reply:
22k -> 29k

Reply:
hmm..
seems like it should be direct dep after all
https://github.com/atlassian/react-beautiful-dnd/issues/845#issuecomment-426963294

Reply:
![Screen Shot 2020-01-03 at 5 52 18 PM](https://user-images.githubusercontent.com/12194969/71717098-ee17fe80-2e51-11ea-80e7-7a5355ba1c5c.png)

phoenix.js 5.62k (gzipped)
babel-runtime 2.83k (gzipped)